ci-operator/jobs/openshift/installer/openshift-installer-master-periodics.yaml

```diff
@@ -0,0 +1,60 @@
+periodics:
```

Comment:
This would normally be a release periodic, we put them together so you can track.  Search for "canary-*" and you should see a bunch of `canary-release-*` jobs.  Move this into there and give it consistent names.

Reply:
Sorry, I see you have a periodic here.  Basically if you have a release periodic you shouldn't do a post submit.  It'll show the same info but in a confusing way.

Reply:
I intended to remove that from the last rebase (my first attempt here). Let me get rid of the post-commit job and just leave the periodic(canary) job.